Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,145 @@
+                return "-";
+            }
+            else
+            {
+                return name;
+            }
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            bool isValidHeightAndWeight = height > 0 && weight > 0;
+            const string heightError = "Height cannot be equal or less than zero, but was ";
+            const string heightErrorForNegative = "Height cannot be less than zero, but was ";
+            const string weightError = "Weight cannot be equal or less than zero, but was ";
+            if (isValidHeightAndWeight)
+            {
+                return weight / (float)Math.Pow(height, 2);
+            }
+            else
+            {
+                if (height <= 0 && weight <= 0)
+                {
+                    DisplayErrorBmi($"{weightError}{weight}.{Environment.NewLine}{heightErrorForNegative}{height}.");
+                }
+                else if (height <= 0)
+                {
+                    DisplayErrorBmi($"{heightError}{height}.");
+                }
+                else
+                {
+                    DisplayErrorBmi($"{weightError}{weight}.");
+                }
+                return -1;
+            }
+        }
+
+        public static void DisplayErrorBmi(string message)
+        {
+            Console.WriteLine($"Failed calculating BMI. Reason:{Environment.NewLine}{message}");
+        }
+
+        public static float PromptFloat(string message)
+        {
+            Console.WriteLine(message);
+
+            string input = Console.ReadLine();
+
+            if (IsValidFloat(input))
+            {
+                return float.Parse(input);
+            }
+            else if (String.IsNullOrEmpty(input))
+            {
+                return 0;
+            }
+            else
+            {
+                return DisplayError(input);
+            }
+        }
+
+        public static int PromptInt(string message)
+        {
+            Console.WriteLine(message);
+
+            string input = Console.ReadLine();
+
+            if (IsValidInt(input))
+            {
+                return int.Parse(input);
+            }
+            else if (String.IsNullOrEmpty(input))
+            {
+                return 0;
+            }
+            else
+            {
+                return DisplayError(input);
+            }
+        }
+
+        public static int DisplayError(string input)
+        {
+            Console.Write($"\"{input}\" is not a valid number.");
+            return -1;
+        }
+
+        public static bool IsValidFloat(string input)
+        {
+            bool isValidNumber = float.TryParse(input, out float number) && (number > 0);
+            return isValidNumber;
+        }
+        public static bool IsValidInt(string input)
+        {
+            bool isNumber = int.TryParse(input, out int number);
+            bool isPositive = (number > 0) ? true : false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971302)